<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="category_lablel">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="category_lineedit">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter a category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="category_submit">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>120</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
